uvicorn/middleware/wsgi.py

```diff
@@ -99,11 +100,11 @@ def __init__(
     async def __call__(
         self, receive: ASGIReceiveCallable, send: ASGISendCallable
     ) -> None:
-        message = await receive()
+        message: HTTPRequestEvent = await receive()
         body = message.get("body", b"")
```

Comment:
Was getting this error:
```
uvicorn/middleware/wsgi.py:103: error: TypedDict "HTTPDisconnectEvent" has no key 'body'  [misc]
uvicorn/middleware/wsgi.py:103: error: TypedDict "WebsocketConnectEvent" has no key 'body'  [misc]
uvicorn/middleware/wsgi.py:103: error: TypedDict "WebsocketReceiveEvent" has no key 'body'  [misc]
uvicorn/middleware/wsgi.py:103: error: TypedDict "WebsocketDisconnectEvent" has no key 'body'  [misc]
uvicorn/middleware/wsgi.py:103: error: TypedDict "LifespanStartupEvent" has no key 'body'  [misc]
uvicorn/middleware/wsgi.py:103: error: TypedDict "LifespanShutdownEvent" has no key 'body'  [misc]
```

Reply:
That's exactly what was bugging me 😅
Most probably `http.disconnect` could get here.
But if I remove the type annotation from `message`, the `body = message.get("body", b"")` line throws out the errors.
Not sure how to resolve that then.


PS. The same goes for `body_message` a couple of lines below that.